cli/src/lib.rs

```diff
@@ -82,6 +82,9 @@ pub enum Command {
         /// Don't install JavaScript dependencies
         #[clap(long)]
         no_install: bool,
+        /// Package Manager to use
+        #[clap(value_enum, long, default_value = "npm")]
```

Comment:
Let's also keep this as `yarn` until v0.32:
```suggestion
        #[clap(value_enum, long, default_value = "yarn")]
```